schematics/src/schematics.consts.ts

```diff
@@ -2,3 +2,5 @@ export const PROJECT_NAME = 'transloco';
 export const NPM_SCOPE = '@ngneat';
 export const LIB_NAME = '@ngneat/transloco';
 export const CONFIG_FILE = 'transloco.config.js';
+export const KEYS_MANAGER_VERSION = '2.2.1';

```

Comment:
what if we update the keys manager version? I wouldn't like updating it over and over...
p.s. moving the script to the keys-manager lib will fix it but I am not sure we like it... (maybe we will do it in the future but currently this is not the solution I am looking for...).

Reply:
If you update keys-manager then you update the version that the schematic installs. It's annoying however it's better to do it this way, than to forcibly run a command like `npm install`

Reply:
Further to that, packages like angular itself hard code the version, or at least they hard code the assumption to use the current version of the cli, as all angular packages are supposed to have the same semver. So instead of doing that, in this case, you would need to keep track of which version to explicitly install. 

Technically you can use `latest` as the version, however I don't think that's good practice to have `latest` in the package.json

Reply:
> p.s. moving the script to the keys-manager lib will fix it but I am not sure we like it... (maybe we will do it in the future but currently this is not the solution I am looking for...).

Moving the script to the keys-manager lib actually makes more sense than keeping it in this lib, as you then don't need to update the version in this repo, and semantically you are adding keys-manager to the project, which is a dev dependency, through `ng add` not `ng g` which is a little confusing anyway.

Reply:
> > p.s. moving the script to the keys-manager lib will fix it but I am not sure we like it... (maybe we will do it in the future but currently this is not the solution I am looking for...).
> 
> Moving the script to the keys-manager lib actually makes more sense than keeping it in this lib, as you then don't need to update the version in this repo, and semantically you are adding keys-manager to the project, which is a dev dependency, through `ng add @ngneat/transloco-keys-manager` instead of `ng g @ngneat/transloco:keys-manager` which is generating something rather than adding something

Sure, but as I said we currently do not want to do it... also, keep in mind it is not an Angular project so I am not sure if it even possible right now... anyway I am looking for a simple solution that detect if the user is using `yarn` as a package manager (I think you can get that from the `angular.json`), and run `yarn` or `npm` accordingly.